### PR TITLE
Feature/create view

### PR DESCRIPTION
**Pull request by larabr**

### CODE REVIEW

File: public/components/freelancer-create.js [hunk 1]
```diff
@@ -81,14 +81,16 @@ class CreationForm extends React.Component {
...
 
-            <label className="category-selector">
-              Job category:
+            <div className="category-selector">
+              <span className="bar"></span>
+              <label>
+                Job Category
+              </label>
               <select ref="category">
                 <option value="selected" disabled>Please select a job category</option>
```
Comment (aronfiechter) [inline public/components/freelancer-create.js]: This does not work, the form displays directly `testCategory1` and does not show the hint.
Also, the category selector looks a bit too large.

File: public/components/freelancer-create.js [hunk 2]
```diff
@@ -81,14 +81,16 @@ class CreationForm extends React.Component {
...
                 {this.props.categories}
               </select>
-
-            </label>
+            </div>
 
             <div className="group">
               <input ref="address" className="address" name="address" type="text" required/>
```
Comment (aronfiechter) [inline public/components/freelancer-create.js]: Should not be required, as in the model. The only required fields should be `name`, `title`, `category`, `email`.

**Review by aronfiechter — CHANGES_REQUESTED** [review]
Too many fields are required in the form; I'd say that the only required ones should be the ones required by the `Freelance` model (plus `category`, which we should mark as required in the model).

Also, `phone` should only accept numbers, with an optional `+` at the beginning.

Other  fields such as `price` and `description` are missing, but maybe you though about adding them later, so it's ok. I don't know how much the product owner wanted to have.

File: public/css/freelancer-create.css [hunk 3]
```diff
@@ -45,6 +45,26 @@
   width: 50%;
 }
 
+/* submit button */
```
Comment (lazy-val) [inline public/css/freelancer-create.css:48]: http://fs5.directupload.net/images/170319/lgkobh8o.png

Same observations as here:
https://github.com/softwareatelier4/Group1/pull/8

File: public/components/freelancer-create.js (hunk 1 above)
Comment (larabr) [reply public/components/freelancer-create.js]: @Spyridox i know, the hint is not supposed to be selectable, i can put it as such but then i have to do a weird check to see if its a valid category

Comment (aronfiechter) [reply public/components/freelancer-create.js]: In most forms if you have a "please select a category" then it is not selectable, but it's the default value of the selector. Why have it if it is never shown?

It should show the hint by default, require the user to change it and not allow to reselect the hint.

File: public/css/freelancer-create.css (hunk 3 above)
Comment (aronfiechter) [reply public/css/freelancer-create.css:48]: Yes, I also had some observations from #8 that I cannot add here.

File: public/components/freelancer-create.js (hunk 2 above)
Comment (larabr) [reply public/components/freelancer-create.js]: k

File: public/css/freelancer-create.css (hunk 3 above)
Comment (larabr) [reply public/css/freelancer-create.css:48]: k

File: public/components/freelancer-create.js (hunk 1 above)
Comment (larabr) [reply public/components/freelancer-create.js]: k

**Pull request merged**